plutus-pab/app/CommandParser.hs

```diff
@@ -141,6 +130,7 @@ migrationParser =
                 str
                 (metavar "DATABASE" <>
                  help "The sqlite database file.")
+        -- TODO: This will need to be 'WithConfig'.
```

Comment:
Please check with @gilligan - being able to run `migrate` without a config file was added to make the VM test easier

Reply:
@j-mueller I see. At the time of this comment I had anticpated needing more information in the config in order to even interact with the database; but now I don't require it, so I'm happy to leave it as-is if it's working.